src/it/resources/org/jenkinsci/account/ui/test-data.ldif

```diff
@@ -0,0 +1,59 @@
+#
+# Dummy user database for testing
+#
+# User 'kohsuke' is admin, user 'alice' is a regular user.
+# Both has the password 'password'
+#
+dn: dc=jenkins-ci,dc=org
+objectClass: top
+objectClass: dcObject
+objectClass: organization
+o: Jenkins users
+dc: Jenkins-Ci
+description: Jenkins users
+
+#dn: cn=admin,dc=jenkins-ci,dc=org
+#objectClass: simpleSecurityObject
+#objectClass: organizationalRole
+#cn: admin
+#description: LDAP administrator
+#userPassword: {SSHA}yI6cZwQadOA1e+/f+T+H3eCQQhRzYWx0
```

Comment:
Shouldn't this commented block be removed?

Reply:
I have a follow up PR https://github.com/timja/account-app/pull/1 which will adjust this so don't worry about it